### PR TITLE
Define 's justify-content to be centered only in large screens

### PR DESCRIPTION
**Pull request by florianeyzat**

### CODE REVIEW

File: src/layout/section/mediaContentSection/__snapshots__/index.unit.tsx.snap [hunk 1]
```diff
@@ -137,8 +137,8 @@ exports[`MediaContentSection should render default MediaContentSection section 1
   -webkit-box-align: center;
   -ms-flex-align: center;
   align-items: center;
-  -webkit-transition: max-width 500ms ease-in, background-color 200ms ease-in;
-  transition: max-width 500ms ease-in, background-color 200ms ease-in;
+  -webkit-transition: max-width 500ms ease-in,background-color 200ms ease-in;
```
Comment (gabrielagabriel) [inline src/layout/section/mediaContentSection/__snapshots__/index.unit.tsx.snap:140]: We gotta decide if spaces or not are the rules for snapshots.
Ping @wakooka

Comment (florianeyzat) [reply src/layout/section/mediaContentSection/__snapshots__/index.unit.tsx.snap:140]: Yes I have an issue with them here..

Comment (florianeyzat) [reply src/layout/section/mediaContentSection/__snapshots__/index.unit.tsx.snap:140]: Basically my snapshots update something different than our CI, I'm going to force push the previous version so we don't see the issue here

Comment (wakooka) [reply src/layout/section/mediaContentSection/__snapshots__/index.unit.tsx.snap]: I sent a message in slack on this: `npm run test -- --clearCache` You shouldn't have the issue after that

**Pull request merged**